DataFormats/BTauReco/src/TaggingVariable.cc

```diff
@@ -127,7 +127,7 @@ const char* const TaggingVariableTokens[] = {
   /* [trackPtRatio]                             = */ "trackPtRatio",
   /* [trackPParRatio]                           = */ "trackPParRatio",
 
-  /* [trackIp2dVal]                             = */ "trackIp2dVal",
+  /* [trackIp2dSig]                             = */ "trackIp2dSig",
```

Comment:
OK, once I manage to have some better connection I will make the checks and the changes you suggested.

Concerning the map I agree that typos can still happen, but the person writing the code will not have to match the token position with the enum position, and since c++11 will have ```{TaggingVariableName, TaggingVariableToken}``` pairs that, in my opinion, are easier to handle. It should also slightly speed up the code. So far the Token to Name matching is done O(N), with a map it would be done in O(LogN), with an unordered map in O(1), but the memory footprint would be larger and the constant time might be longer than LogN for few entries.